data/outfits/god.txt

```diff
@@ -1,36 +1,86 @@
 outfit "Ruin-The-Fun God Mode"
 	"display name" "RTF God Mode"
 	category `RTF God Outfits`
-	cost 0
 	thumbnail "outfit/quarg quantum shield generator"
-	"mass" 0
 	hull 1000000
 	"hull repair rate" 1000000
-	"hull energy" 0
 	shields 1000000
 	"shield generation" 1000000
-	"cooling" 1000000
-	"ion resistance" 1000000
+	"high shield permeability" 0
+	"low shield permeability" 0
 	"disruption resistance" 1000000
+	"ion resistance" 1000000
+	"scramble resistance" 1000000
+	"slowing resistance" 1000000
 	"discharge resistance" 1000000
+	"corrosion resistance" 1000000
+	"leak resistance" 1000000
+	"burn resistance" 1000000
+	"piercing resistance" 1000000
+	"disruption protection" 1000000
+	"energy protection" 1000000
+	"force protection" 1000000
+	"fuel protection" 1000000
+	"heat protection" 1000000
+	"hull protection" 1000000
+	"ion protection" 1000000
+	"scramble protection" 1000000
 	"piercing protection" 1000000
+	"shield protection" 1000000
 	"slowing protection" 1000000
 	"discharge protection" 1000000
 	"corrosion protection" 1000000
 	"leak protection" 1000000
 	"burn protection" 1000000
-	bunks 1000000
-	description "Absolutely illegal."
+	description "Protect your ship against almost anything."
+
+outfit "Ruin-The-Fun God Scanner"
+	"display name" "RTF God Scanner"
+	category "RTF God Outfits"
+	thumbnail "outfit/outfit scanner"
+	"tactical scan power" 100000000
+	"asteroid scan power" 100000000
+	"outfit scan power" 100000000
+	"cargo scan power"  100000000
+	"outfit scan efficiency" 1000000
+	"cargo scan efficiency" 1000000
+	description "Scans far and wide."
+
+outfit "Ruin-The-Fun God Engine"
+	"display name" "RTF God Engine"
+	category "RTF God Outfits"
+	thumbnail "outfit/tiny atomic thruster"
+	"mass" 1000000
+	"inertia reduction" 10000
+	"drag reduction" 10000
+	"thrust" 50
+	"reverse thrust" 50
+	"flare sprite" "effect/atomic flare/tiny"
+		"frame rate" 14
+	"flare sound" "atomic tiny"
+	"reverse flare sprite" "effect/atomic flare/tiny"
+		"frame rate" 14
+	"reverse flare sound" "atomic tiny"
+	description "Runs really fast."
 
+outfit "Ruin-The-Fun God Steering"
+	"display name" "RTF God Steering"
+	category "RTF God Outfits"
+	thumbnail "outfit/tiny atomic steering"
+	"mass" 1000000
+	"inertia reduction" 10000
+	"drag reduction" 10000
+	"turn" 1250
+	"steering flare sprite" "effect/atomic flare/tiny"
+		"frame rate" 14
+	"steering flare sound" "atomic tiny"
+	description "Turns really fast."
```

Comment:
I have merged the two outfits for better inertia and drag reduction.

Reply:
Stats are still fine, I tested it on an Albatross, the one from the quick start, but I had to remove already installed steering otherwise it would be uncontrollable. But it had powerful ones already installed.

If that is a problem, I think `inertia reduction` is optional, the engine can still be balanced using `mass` and `turn`. `drag reduction` can stay, I think it only affects max speed.

Reply:
There is also `acceleration multiplier` and `turn multiplier`, maybe it's more predictable than `inertia reduction`.
I'm fine with the engine right now, that's just an idea.

Reply:
Good:
![image](https://github.com/Pshy0/es-ruin-the-fun/assets/44928041/db21b9f6-7e51-4956-910c-371d7569674e)